js/components/Tabs.js

```diff
@@ -1,5 +1,6 @@
+/* globals gThreadClient */
```

Comment:
was tempted to add this to the list of globals in eslintrc, but figured, we'd be refactoring soon